tests/integration/kubernetes/lib.sh

```diff
@@ -251,11 +250,11 @@ print_node_journal() {
 	shift 2
 	local img="quay.io/prometheus/busybox"
 
-	kubectl debug --image "$img" -q -i "node/${node}" \
```

Comment:
Hi @BbolroC !

Never occurred to me that the debugger pod could end prematurely. Good catch!

Reply:
Yeah, this happens frequently on s390x (even yesterday. see https://github.com/kata-containers/kata-containers/actions/runs/10981530559/job/30489117743, with two false negatives). 

I initially thought that it was an environmental issue on the platform. But after looking at #10081, I realised that it could be related to how we are handling `kubectl debug` in our tests. 

My goal is to run a dedicated `custom-node-debugger` pod first, which was set up in #10232, and use `kubectl exec` on the pod until the test is complete. 

I'm hoping this will resolve the issues around `kubectl debug`. 😉